### ISSUE TITLE
Specify config Json in the created Container

### ISSUE THREAD

**Issue by amihaiemil** [amihaiemil/docker-java-api#133]
When creating the container, [here](https://github.com/amihaiemil/docker-java-api/blob/2965aee5849c793c936c439364934e84b6a9ff58/src/main/java/com/amihaiemil/docker/RtContainers.java#L120), we should pass it a JsonObject containing both the ``container`` parameter (the JsonObject used to create it) and the response JsonObject (which contains only the Id and Warnings).

This is because later we may need to know how the Container was created (e.g. if it has ``TTY`` enabled or not), so we should be able to spare some HTTP calls.

**Comment by amihaiemil** [comment]
@llorllale Do you want to do this?

**Comment by 0crat** [comment]
Job [#133](https://github.com/amihaiemil/docker-java-api/issues/133) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
The job [#133](https://github.com/amihaiemil/docker-java-api/issues/133) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/1590771389); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by llorllale** [comment]
@0crat waiting on review/merge for PR #137

**Comment by 0crat** [comment]
> @0crat waiting on review/merge for PR #137  ([here](https://github.com/amihaiemil/docker-java-api/issues/133#issuecomment-396610456))

@llorllale The impediment for [#133](https://github.com/amihaiemil/docker-java-api/issues/133) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#133](https://github.com/amihaiemil/docker-java-api/issues/133) is now out of scope
